packages/ra-ui-materialui/src/detail/Create.tsx

```diff
@@ -1,18 +1,13 @@
 import * as React from 'react';
-import { Children, cloneElement, useMemo } from 'react';
 import PropTypes from 'prop-types';
-import Card from '@material-ui/core/Card';
-import { makeStyles } from '@material-ui/core/styles';
-import classnames from 'classnames';
 import {
+    CreateContextProvider,
     useCheckMinimumRequiredProps,
     useCreateController,
-    SideEffectContext,
-    CreateControllerProps,
 } from 'ra-core';
 
-import TitleForRecord from '../layout/TitleForRecord';
 import { CreateProps } from '../types';
+import { CreateView } from '.';
```

Comment:
I don't like these, they can create circular preferences. Prefer importing from the file is it's in the same directory.